modules/learn/pages/clusters-and-availability/xdcr-filtering.adoc

```diff
@@ -163,3 +170,72 @@ image::xdcr/filter-replication-diagram-7.png[,720,align=left]
+Note, however, that this would also prevent the replication of future source-deletions of type _B_ documents: therefore, the creation of an entirely new source collection for type _B_ documents might be required, by means of _migration_, prior to the creation of replication _2_.
+
+=== Deletion Filters and Migration
+
+The appropriate configuring of deletion filters is critically important in cases of _migration_ where documents are being assigned to newly created collections, with their source collection (often the default collection of a legacy bucket) intended subsequently to be dropped.
+For example, the following sequence results in loss of data:
+
+. The default collection of a legacy bucket is determined to contain only documents that are either of type _A_ or of type _B_.
+
+. Migration is configured to replicate documents of type _A_ from the source, default collection to a new, target collection, named _A_; and documents of type _B_ to another new, target collection, named _B_.
+Deletion filters are left at their default settings.
+
+. Migration proceeds.
+Eventually, all type _A_ documents exist both in the source collection, and in the new, target collection _A_; and all type _B_ documents exist both in the source collection, and in the new, target collection _B_.
+
+. The source, default collection is dropped; and all its data thereby deleted.
+
+Here, the (incorrectly) expected result has been that all documents will continue to exist; in the new, target collections, _A_ and _B_.
+However, since the migration was not deleted prior to the deletion of the source data, and since the default settings of the deletion filters specified that document-deletions should be replicated; the actual result is that all documents from the target collections _A_ and _B_ have been deleted, along with the source, default collection, and all its data.
+
+==== Guarding Against Accidental Data-Loss during Migration
+
+Either of the following approaches can be used to ensure that no migrated data is lost:
+
+* Configure deletion filters to prohibit the replication of deletions.
+This ensures that only documents and their mutations are replicated to their new collection.
++
+Note, however, that if read-write application-access continues to be granted to the source collection during the life of the migration, application-deletions that occur on the source are not replicated to the target collection; eventually rendering source and target collections inconsistent.
+
+* Keep deletion filters at their default setting, to permit the replication of deletions.
+When the migration is judged to have completed, delete the migration _prior to_ the deletion of any source data.
+Then, once the migration is deleted, delete source data as appropriate.
```

Comment:
Additional note as a reminder - if the replication is re-created after this point without a deletion filter, and the tombstones from the deletes have not been purged, then the deletes will be replicated to the target bucket.

Reply:
I've added a note to this effect.